build.gradle

```diff
@@ -49,6 +49,11 @@ if (hasProperty('cfLocal')) {
   def cfHome = String.valueOf(System.getenv('CHECKERFRAMEWORK'))
```

Comment:
It's too far away to comment on, so let me comment here:
````
  // This dependency is used by the application.
  implementation libs.guava
````

The project setup is a bit odd - there is no actual application, only the test.
If it's easy to do, can you change the whole project into just the conformance runner?
If that's difficult, what is the above `implementation` dependency for?

Reply:
This dependency was unnecessary.